test/scripts/base-host-check.sh

```diff
@@ -169,4 +169,12 @@ if (( $# > 0 )); then
     if jq -e '.blueprint.enabled_modules' "${config}" || jq -e '.blueprint.packages[] | select(.name | startswith("@") and contains(":")) | .name' "${config}"; then
         check_modularity "${config}"
     fi
+
+    if jq -e '.blueprint.customizations.hostname' "${config}"; then
+        [[ "$(hostname)" == "$(jq -r '.blueprint.customizations.hostname' "${config}")" ]]
+    fi
+
+    if jq -e '.blueprint.customizations.kernel.name' "${config}"; then
+        [[ "$(uname -s)" == "$(jq -r '.blueprint.customizations.kernel.name' "${config}")" ]]
```

Comment:
This is probably not the correct approach.

```console
[root@vm-10-0-184-196 ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux release 9.7 Beta (Plow)
[root@vm-10-0-184-196 ~]# uname -s
Linux
```

Reply:
yeah I don't know what I was thinking.  It turns out this is pretty tricky to check.  I gave it a shot though.